Makefile

```diff
@@ -37,6 +37,8 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # oraclecloud/upgradeoperatorsdk-bundle:$VERSION and oraclecloud/upgradeoperatorsdk-catalog:$VERSION.
 IMAGE_TAG_BASE ?= iad.ocir.io/oracle/oci-service-operator
 
+INDEX_IMG ?= iad.ocir.io/idotidhmwg5o/operator-framework/upstream-opm-builder:latest
```

Comment:
We might not want to push this and target bundle-deploy, bundle-undeploy to public